inception-concept-linking/src/main/java/de/tudarmstadt/ukp/inception/conceptlinking/recommender/NamedEntityLinkerTraits.java

```diff
@@ -19,6 +19,9 @@
 
 import java.io.Serializable;
 
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
```

Comment:
Dependency needs to be added to the POM - build seems to fail because of this.

Reply:
Done